### ISSUE TITLE
Unable to retrieve Socure artifacts from jitpack

### ISSUE THREAD

**Issue by DougMcTuro** [socure-inc/socure-sigmadevice-sdk-android#16]
Gradle sync produces the following error:
`Failed to resolve: com.github.socure-inc:device-risk-android-sdk:3.0.0`

At command line, `curl https://jitpack.io/com/github/socure-inc/device-risk-android-sdk/3.0.0/device-risk-android-sdk-3.0.0.pom` produces `Repo not found or no token provided`

**Comment by rburunkayaSocure** [comment]
Hi @DougMcTuro ,

The issue may related to downtime on JitPack site but we just tested and are able to download artifact right now. Please make sure you have added `maven { url 'https://jitpack.io' }` in place.

I also want to inform you that we are planning to release Device Android SDKs version 4.0.0 and start hosting SigmaDevice SDKs on Socure owned Maven repository. We are going to provide an update when new SigmaDevice SDKs arrives.

Kind regards,

**Comment by hjdealba96** [comment]
Hello @rburunkayaSocure,

It's still failing on our end with `com.github.socure-inc:socure-docv:3.2.0`:

```
* What went wrong:
Execution failed for task ':app:dataBindingMergeDependencyArtifactsRelease'.
> Could not resolve all files for configuration ':app:releaseCompileClasspath'.
   > Could not resolve com.github.socure-inc:socure-docv:3.2.0.
     Required by:
         project :app
      > Could not resolve com.github.socure-inc:socure-docv:3.2.0.
         > Could not get resource '[https://jitpack.io/com/github/socure-inc/socure-docv/3.2.0/socure-docv-3.2.0.pom'](https://jitpack.io/com/github/socure-inc/socure-docv/3.2.0/socure-docv-3.2.0.pom).
            > Could not GET '[https://jitpack.io/com/github/socure-inc/socure-docv/3.2.0/socure-docv-3.2.0.pom'](https://jitpack.io/com/github/socure-inc/socure-docv/3.2.0/socure-docv-3.2.0.pom). Received status code 401 from server: Unauthorized
```

On the other hand, do you have an ETA for the release of v4.0.0?

**Comment by rburunkayaSocure** [comment]
Hey @hjdealba96 

Just tested `com.github.socure-inc:socure-docv:3.2.0` and able to download artifacts. The issue may be related to JitPack but we started hosting Socure DocV SDKs on Socure owned Maven repository with version 4.3.1. Please see more details about this repository here -> https://github.com/socure-inc/socure-docv-sdk-android

Socure SigmaDevice SDKs v4.0.0 is going to be released soon. I am going to notify you when it is in production.

Kind regards,

**Comment by rburunkayaSocure** [comment]
Hi @hjdealba96 

We released Sigma Device Android SDKs v4.0.1 and started hosting SigmaDevice Android SDKs on Socure owned Maven repository.

Kind regards,
